server/helpers/constants.js

```diff
@@ -120,3 +120,9 @@ export const ACTIVE_PORTFOLIO_OFFER = [
 ];
 
 export const REFERRAL_PERCENTAGE = 1.5;
+
+export const BADGE_ACCESS_LEVEL = {
+  ALL: 0,
+  USER: 1,
```

Comment:
I think it is better to link this directly with the user and vendor role.
```suggestion
  USER: USER_ROLE.USER,
```